tap_sftp/client.py

```diff
@@ -146,8 +167,17 @@ def get_files(self, prefix, search_pattern, modified_since=None):
         return sorted_files
 
     def get_file_handle(self, f):
-        """ Takes a file dict {"filepath": "...", "last_modified": "..."} and returns a handle to the file. """
-        return self.sftp.open(f["filepath"], 'rb')
+        """ Takes a file dict {"filepath": "...", "last_modified": "..."}
+        -> returns a handle to the file.
+        -> raises error with appropriate logger message """
+        try:
+            return self.sftp.open(f["filepath"], 'rb')
+        except OSError as e:
+            if "Permission denied" in str(e):
+                LOGGER.info("Skipping %s file because you do not have enough permissions.", f["filepath"])
+            else:
+                LOGGER.info("Skipping %s file because it is unable to be read.", f["filepath"])
+            raise OSError
```

Comment:
We want to re-raise the original exception here which can be done by just using `raise`.
```suggestion
            raise
```

Reply:
Done